src/components/ApeCheckbox/ApeCheckbox.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
```

Comment:
what was your reasoning for creating this with Material-UI rather than Stitches?

Reply:
No Reasoning actually. When I started working on this issue, I wasn't totally aware of the migration to stitches. But I will change that to stitches too.